app/presenters/two_factor_authentication/piv_cac_selection_presenter.rb

```diff
@@ -7,5 +7,13 @@ def method
     def security_level
       I18n.t('two_factor_authentication.two_factor_choice_options.more_secure_label')
     end
+
+    def disabled?
+      !user.nil? && user.piv_cac_configurations.any?
```

Comment:
ditto about the null-safe navigator
```suggestion
      user&.piv_cac_configurations&.any?
```